src/mergerepo_c.c

```diff
@@ -220,7 +114,10 @@ static GOptionEntry cmd_entries[] =
 
     // -- Options related to Koji-mergerepos behaviour
     { "koji", 'k', 0, G_OPTION_ARG_NONE, &(_cmd_options.koji),
-       "Enable koji mergerepos behaviour.", NULL},
+       "Enable koji mergerepos behaviour. (Optionally select simple mode with: --simple)", NULL},
+    { "simple", 0, 0, G_OPTION_ARG_NONE, &(_cmd_options.koji_simple),
+       "Enable koji specific simple merge mode where we keep even packages with "
+       "identical NEVRAs. (Doesn't work without --koji)", NULL},
```

Comment:
There is new text `(Doesn't work without --koji)` added.
There is another option `blocked` with text `Only works with combination with --koji/-k.`
For consistency I prefer the same message for both options.

Reply:
Pushed an update with reformatted option text.